client/components/tile-grid/style.scss

```diff
@@ -85,11 +90,13 @@
 			width: 8px; //Match the size of the cta copy
 			height: 8px; //Match the size of the cta copy
 			position: absolute;
-			top: 20px;
+			top: 50%;
 			right: 15px;
+			margin-top: -6px;
```

Comment:
This works here, but you could also use a transform property to move it as well `translateY( -50% )` or something like that. No change needed, but handy for things where you don't know the height.

Reply:
Sure, I actually wanted to do it this way, but it would still require a `margin-top: -2px` because of the `border-top` that we have. That's why I went with the `margin`